solana/ts/sdk/ntt.ts

```diff
@@ -891,15 +958,98 @@ export class NTT {
    * (i.e. the program is initialised), the mint is derived from the config.
    * Otherwise, the mint must be provided.
    */
-  async custodyAccountAddress(configOrMint: Config | PublicKey): Promise<PublicKey> {
+  async custodyAccountAddress(configOrMint: Config | PublicKey, tokenProgram = splToken.TOKEN_PROGRAM_ID): Promise<PublicKey> {
     if (configOrMint instanceof PublicKey) {
-      return associatedAddress({ mint: configOrMint, owner: this.tokenAuthorityAddress() })
+      return splToken.getAssociatedTokenAddress(configOrMint, this.tokenAuthorityAddress(), true, tokenProgram)
     } else {
-      return associatedAddress({ mint: await this.mintAccountAddress(configOrMint), owner: this.tokenAuthorityAddress() })
+      return splToken.getAssociatedTokenAddress(configOrMint.mint, this.tokenAuthorityAddress(), true, configOrMint.tokenProgram)
     }
   }
 }
 
 function exhaustive<A>(_: never): A {
   throw new Error('Impossible')
 }
+
+/**
+ * TODO: this is copied from @solana/spl-token, because the most recent released
```

Comment:
Might be good to also capture this in a GH Issue so it doesn't get lost

Reply:
opened an issue https://github.com/wormhole-foundation/example-native-token-transfers/issues/417 thanks!